Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb

```diff
@@ -0,0 +1,1438 @@
+{
```

Comment:
<p>Could you make sure to follow the guidelines in #46 </p><p>I also see the Markdown <code>#</code>  have not been used properly, you have missed spaces.. I will perform a detailed review on this soon.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I have updated it.